src/core.js

```diff
@@ -104,8 +104,11 @@ async function sync(cloneUrl, localDirectory, {ref, checkout}) {
   }
 
   await git.cwd(localDirectory);
-  log.info(`> Fetching ${ref}...`);
-  await git.fetch('origin', ref);
+  const remoteName = cloneName || 'origin';
+  await git.removeRemote(remoteName).catch((error) => {}); // eslint-disable-line no-unused-vars
+  await git.addRemote(remoteName, cloneUrl);
```

Comment:
fwiw these strings are like `"username/reponame"`, and git is totally cool with slashes in remote names, afaict.

Reply:
We also shouldn't have to validate because GitHub users shouldn't be able to create a branch with invalid characters to begin with! Should fail further upstream, not at the `stage-ci` level 🚀